NitroxServer/Communication/LiteNetLib/PortCheckerSupport.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Net;
+using LiteNetLib.Layers;
+using LiteNetLib;
+using System.Collections.Generic;
+using LiteNetLib.Utils;
+using System.Net.Sockets;
+namespace NitroxServer.Communication.LiteNetLib;
+
+public class PortCheckerSupport : PacketLayerBase
+{
+    public static bool active;
+    public NetManager netManager;
+    public PortCheckerSupport() : base(0)
+    {
+        active = false;
+    }
+    
+    public override void ProcessInboundPacket(ref IPEndPoint endPoint, ref byte[] data, ref int length)
+    {
+        if (active)
+        {
+            Log.Info("Incoming packet");
+            netManager.SendUnconnectedMessage(data, endPoint);
```

Comment:
Recommend using UdpClient for sending raw data back as LiteNetLib does its own mutations on the data before sending.

Reply:
Implemented.